### PR TITLE
Field oriented and rotation-less autonomous

### PR DESCRIPTION
**Pull request by BrandonWerbel**

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java
Comment (ErikBoesen) [inline src/main/java/frc/robot/RobotContainer.java:171]: Would be nice to declare this as a constant so it's clear what 5 means

File: src/main/java/frc/robot/RobotContainer.java
Comment (ErikBoesen) [inline src/main/java/frc/robot/RobotContainer.java:173]: Same thing with this 5 and 12

**Review by ErikBoesen — APPROVED** [review]
LGTM but wait on Tim's review before merging

**Comment by BrandonWerbel** [comment]
@ArchdukeTim It'd be awesome if you could look over the PR, but I'm probably going to merge it at the end of today if you don't.

File: src/main/java/frc/robot/Robot.java
Comment (ArchdukeTim) [inline src/main/java/frc/robot/Robot.java:54]: lol is this so you can push the robot?

Comment (BrandonWerbel) [reply src/main/java/frc/robot/Robot.java:54]: Yeahhh, I'm sure there's a prettier solution, but this works

File: src/main/java/frc/robot/subsystems/SwerveDriveSubsystem.java
Comment (ArchdukeTim) [inline src/main/java/frc/robot/subsystems/SwerveDriveSubsystem.java:76]: you should use the drive method you have so that any time you want to make logic changes from moduleStates on you only have to once

File: src/main/java/frc/robot/subsystems/SwerveDriveSubsystem.java
Comment (ArchdukeTim) [inline src/main/java/frc/robot/subsystems/SwerveDriveSubsystem.java:98]: Maybe make `resetEncoders` a method on `WheelSubsystem`

File: src/main/java/frc/robot/subsystems/SwerveDriveSubsystem.java
Comment (ArchdukeTim) [inline src/main/java/frc/robot/subsystems/SwerveDriveSubsystem.java:116]: These could also be part of the subsystem instead of the swerve drive system, no?

File: src/main/java/frc/robot/subsystems/WheelSubsystem.java
Comment (ArchdukeTim) [inline src/main/java/frc/robot/subsystems/WheelSubsystem.java:46]: Make these magic numbers constants

File: src/main/java/frc/robot/subsystems/WheelSubsystem.java
Comment (ArchdukeTim) [inline src/main/java/frc/robot/subsystems/WheelSubsystem.java:53]: so it just always runs forward?

File: src/main/java/frc/robot/subsystems/WheelSubsystem.java
Comment (ArchdukeTim) [inline src/main/java/frc/robot/subsystems/WheelSubsystem.java:102]:
```suggestion
    public double getTargetSpeed() {
```

File: src/main/java/frc/robot/subsystems/WheelSubsystem.java
Comment (ArchdukeTim) [inline src/main/java/frc/robot/subsystems/WheelSubsystem.java:112]: What's wrong with `turningEncoder.getPosition`?

File: src/main/java/frc/robot/subsystems/WheelSubsystem.java
Comment (ArchdukeTim) [inline src/main/java/frc/robot/subsystems/WheelSubsystem.java:37]: you should make these variable names have `nt` or something in them because it's very confusing to read

File: src/main/java/frc/robot/subsystems/SwerveDriveSubsystem.java
Comment (BrandonWerbel) [reply src/main/java/frc/robot/subsystems/SwerveDriveSubsystem.java:116]: If we put this in the wheelSubsystem, we can't keep each networkTableEntry separate.

File: src/main/java/frc/robot/subsystems/WheelSubsystem.java
Comment (BrandonWerbel) [reply src/main/java/frc/robot/subsystems/WheelSubsystem.java:53]: FeedForward was enough for accurate velocity, so we didn't bother with PID.

File: src/main/java/frc/robot/subsystems/WheelSubsystem.java
Comment (BrandonWerbel) [reply src/main/java/frc/robot/subsystems/WheelSubsystem.java:112]: Interestingly, that is not an actual function! The documentation for getAbsolutePosition says 
```"getAbsolutePosition()` - getPositionOffset() will give an encoder absolute position relative to the last reset. This could potentially be negative, which needs to be accounted for."```

**Pull request merged**
